pjlib/src/pj/ssl_sock_imp_common.c

```diff
@@ -339,7 +343,7 @@ static pj_bool_t on_handshake_complete(pj_ssl_sock_t *ssock,
 	/* Decrement ref count of parent and reset parent (we don't need it
 	 * anymore, right?).
 	 */
-	if (ssock->parent->param.grp_lock) {
+	if (ssock->parent && ssock->parent->param.grp_lock) {
```

Comment:
If `ssock->parent` is NULL it will also cause issues when calling the callaback `on_accept_complete/complete2()` above.